src/ctap/storage.rs

```diff
@@ -420,6 +523,13 @@ impl From<StoreError> for Ctap2StatusCode {
     }
 }
 
+fn should_reset<'a>(entry: &StoreEntry<'a>) -> bool {
```

Comment:
Nit: IIRC we can use an anonymous lifetime for this kind of use case, i.e. `fn should_reset(entry: &StoreEntry<'_>) -> bool`.

Reply:
Indeed, the name is not needed. Done.